libs/sdk-backend-tiger/src/backend/workspace/measures/measureExpressionTokens.ts

```diff
@@ -15,8 +17,10 @@ export interface IExpressionToken {
 
 const REMOVE_BRACKETS_REGEXP = /[[\]{}]/g;
 const TOKEN_TYPE_REGEXP_PAIRS: Array<[ExpressionTokenType, RegExp]> = [
-    ["text", /^[^#{}[\]"]+/],
+    ["text", /^[^#{}[\]"0-9()]+/],
     ["quoted_text", /^"(?:[^"\\]|\\"|\\\\.)*"/],
+    ["number", /^[0-9]+/],
```

Comment:
What about decimals?

Reply:
Ha, my fault, its look like that we have also bug in meditor ....